sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ConnectionPolicy.java

```diff
@@ -669,7 +669,7 @@ public String toString() {
             ", minConnectionPoolSizePerEndpoint=" + minConnectionPoolSizePerEndpoint +
             ", openConnectionsConcurrency=" + openConnectionsConcurrency +
             ", aggressiveWarmupConcurrency=" + aggressiveWarmupConcurrency +
-            ", pendingAcquireMaxCount=" + this.pendingAcquireMaxCount != null ? String.valueOf(this.pendingAcquireMaxCount) : "DEFAULT" +
+            ", pendingAcquireMaxCount=" + (this.pendingAcquireMaxCount != null ? String.valueOf(this.pendingAcquireMaxCount) : "DEFAULT") +
```

Comment:
If you wanted something a bit cleaner

```suggestion
            ", pendingAcquireMaxCount=" + Objects.toString(this.pendingAcquireMaxCount, "DEFAULT") +
```

Reply:
updated